### PR TITLE
Added Helm sub-chart for Meshery Adapter for Nighthawk

### PR DESCRIPTION
**Pull request by MeenuyD**
This commit adds a new Helm chart for the Meshery Nighthawk adapter in the v0.8 release. The Helm chart is created based on an existing adapter chart, with all adapter-specific references replaced with references to the Meshery Nighthawk adapter. 
#10370

### CODE REVIEW

**Comment by leecalcote** [comment]
That was fast. 🏃

**Comment by leecalcote** [comment]
An item to resolve - https://github.com/meshery/meshery/actions/runs/7997986435/job/21843417556?pr=10375#step:4:7

**Comment by MeenuyD** [comment]
Can you please tell me which command should I run to solve the  https://github.com/meshery/meshery/actions/runs/7997986435/job/21843417556?pr=10375#step:4:7

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.7.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+# TODO: Need the confirm the image tag.
+appVersion: stable-latest
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml:27]: @MeenuyD We only uploaded [meshery/meshery-nighthawk:edge-latest](https://hub.docker.com/r/meshery/meshery-nighthawk/tags) currently, and there may not be a stable version in a short time. Can we use `edge-latest` at first? // @leecalcote

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.MD [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| annotations | object | `{}` |  |
+| env | object | `{}` |  |
+| fullnameOverride | string | `""` |  |
+| image.pullPolicy | string | `"Always"` |  |
+| image.repository | string | `"layer5/meshery-nighthawk:stable-latest"` |  |
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.MD:23]: Ditto, and url should be `meshery/meshery-nighthawk:edge-latest`

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/values.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
+# Default values for meshery-nighthawk.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: meshery/meshery-nighthawk:stable-latest
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/values.yaml:8]: Ditto.

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.MD [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# meshery-nighthawk
+
+![Version: 0.5.0](https://img.shields.io/badge/Version-0.5.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: stable-latest](https://img.shields.io/badge/AppVersion-stable--latest-informational?style=flat-square)
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.MD:3]: Ditto

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml (hunk 1 above)
Comment (leecalcote) [reply install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml:27]: Makes sense to me.

**Comment by MeenuyD** [comment]
Ok

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.MD [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# meshery-nighthawk
+
+![Version: 0.5.0](https://img.shields.io/badge/Version-0.5.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: edge-latest](https://img.shields.io/badge/AppVersion-stable--latest-informational?style=flat-square)
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.MD:3]:
```suggestion
![Version: 0.5.0](https://img.shields.io/badge/Version-0.5.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: edge-latest](https://img.shields.io/badge/AppVersion-edge--latest-informational?style=flat-square)
```

Comment (gyohuangxin) [reply install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.MD:3]: Another place needs to be replaced.

**Comment by MeenuyD** [comment]
Thank you @gyohuangxin and @leecalcote for correcting me

**Comment by leecalcote** [comment]
> Can you please tell me which command should I run to solve the https://github.com/meshery/meshery/actions/runs/7997986435/job/21843417556?pr=10375#step:4:7

@MeenuyD type `make` and hit `enter`. You'll see a complete list of make targets available.

here are relevant commands:

```
helm-docs           Generate all Meshery Helm Chart documentation in markdown format.
helm-lint           Lint all of Meshery's Helm Charts
helm-meshery-docs   Generate Meshery Server and Adapters Helm Chart documentation in markdown format.
helm-meshery-lint   Lint Meshery Server and Adapter Helm Charts
helm-operator-docs  Generate Meshery Operator Helm Chart documentation in markdown format.
helm-operator-lint  Lint Meshery Operator Helm Chart
```

**Comment by leecalcote** [comment]
@MeenuyD how did those commands work for you?

**Comment by MeenuyD** [comment]
I am working on that

**Comment by gyohuangxin** [comment]
@MeenuyD Do you have any update?

**Pull request closed without merging**